extensions/wikia/CommunityPage/scripts/CommunityPageBenefitsPageviews.js

```diff
@@ -0,0 +1,37 @@
+require([
+	'wikia.pageviewsInSession',
+	'CommunityPageBenefitsModal',
+	'wikia.tracker',
+	'wikia.cookies'
+], function (pageviews, modal, tracker, cookies) {
+	'use strict';
+
+	var wikiaDomain = mw.config.get('wgDevelEnvironment') ? '.wikia-dev.com' : '.wikia.com',
+		// This cookie is checked in CommunityPageSpecialHooks::onBeforePageDisplay to avoid unnecessary script loading
+		modalShownCookieName = 'cpBenefitsModalShown',
+		modalShownExpirationTime = 2592000; // 30 days
+
+
+	function init() {
+		if (pageviews.getPageviewsCount() === 4) {
+			modal.open();
+			setModalShownCookie();
+			trackModalImpression();
+		}
+	}
+
+	function setModalShownCookie() {
+		cookies.set(modalShownCookieName, 1, { domain: wikiaDomain, expires: modalShownExpirationTime });
```

Comment:
I'd also make sure to set the path on the cookies to `mw.config.get('wgCookiePath')`.